.github/workflows/test.yml

```diff
@@ -31,7 +31,8 @@ jobs:
         # criu repo
         sudo add-apt-repository -y ppa:criu/ppa
         # apt-add-repository runs apt update so we don't have to
-        sudo apt -q install libseccomp-dev criu
+        sudo apt -q install criu gperf build-essential
+        sudo ./script/install-libseccomp.sh x86_64-linux-gnu
```

Comment:
It looks like ubuntu 20.04 have libseccomp 2.5.1 in updates now, so this commit may no longer be necessary.

Reply:
Yeap, and the updates is used in the github CI. So removing this chunk, thanks!